lib/OpenCloud/Database/Resource/Instance.php

```diff
@@ -190,13 +201,21 @@ protected function createJson()
             );
         }
 
-        return (object) array(
-            'instance' => (object) array(
+        $out = array(
+            'instance' => array(
                     'flavorRef' => $this->flavor->links[0]->href,
                     'name'      => $this->name,
                     'volume'    => $this->volume
                 )
         );
+
+        if (isset($this->backupRef)) {
+            $out['instance']['restorePoint'] = array(
+                'backupRef' => $this->backupRef
+            );
+        }
+
+        return (object) $out;
     }
 
     /**
```

Comment:
Ok, added. I did not add any checks for required params as they are checked in `Backup::create`. Is that acceptable?